server/src/test/java/io/crate/integrationtests/ColumnPolicyIntegrationTest.java

```diff
@@ -368,10 +371,9 @@ public void testUpdateNewColumnTableDefault() throws Exception {
         execute("refresh table dynamic_table");
 
         waitForMappingUpdateOnAll("dynamic_table", "name");
-        execute("select * from dynamic_table");
```

Comment:
Shouldn't the behavior here stay the same?

Reply:
I was not sure if the update statement had an order. `execute("update dynamic_table set name='Trillian', good=true where score > 0.0");` 
The ordering is lost by the time that the mapping's positions get fixed. Will look into it Thanks.

Reply:
resolved by [ecbbedc](https://github.com/crate/crate/pull/12754/commits/ecbbedcca02059ace6a430de023da9ae89d2f7c9), [a551305](https://github.com/crate/crate/pull/12754/commits/a5513050c0df28c8b00351f5a05dee524dda7ef0).
To briefly explain, I brought back the column position calculations and used the outputs as 'column order' which help generate column positions.